Authentication-System/authen/app/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib import admin
+from django.urls import path,include
+from .views import*
+urlpatterns = [
+    path('',signup),
+    path('signup/',signup),
+    path('login/',userlogin),
```

Comment:
Please use snake_case when working in python for all variable and functions name and use Sentence case for class names. it should be "user_login"

Reply:
updated it now all all variable and functions name are snake_case and Class name are Sentence case